WooCommerce/Classes/ViewRelated/Dashboard/MyStore/PeriodDataViewController.swift

```diff
@@ -38,8 +38,27 @@ class PeriodDataViewController: UIViewController {
     @IBOutlet private weak var chartAccessibilityView: UIView!
 
     private var lastUpdatedDate: Date?
-    private var yAxisMinimum: String = Constants.chartYAxisMinimum.humanReadableString()
-    private var yAxisMaximum: String = ""
+
+    private var revenueItems: [Double] {
+        return orderStats?.items?.map({ $0.grossSales }) ?? []
+    }
+
+    private var yAxisMinimum: String {
+        guard let orderStats = orderStats else {
+            return ""
+        }
+        let min = revenueItems.min() ?? 0
+        return CurrencyFormatter().formatHumanReadableAmount(String(min), with: orderStats.currencyCode) ?? ""
+    }
+
+    private var yAxisMaximum: String {
+        guard let orderStats = orderStats else {
+            return ""
+        }
+        let max = revenueItems.max() ?? 0
+        return CurrencyFormatter().formatHumanReadableAmount(String(max), with: orderStats.currencyCode) ?? ""
+    }
+
```

Comment:
I think we should group these together with the `xAxisMinimum` and `xAxisMaximum` properties.